### DISCUSSION TITLE
Save DNN Page link

### DISCUSSION THREAD

**Discussion by maduranga001** [DNNCommunity/DNN.FormAndList#114] [Q&A]
DNN 9.13.4
FNL 6.5.8

I use URL field to select existing DNN pages. But it saved the tab ID. How can I save the page link to the database UserDefinedData table? I need to get this and show it back via an SQL to display in a chart. So I need it to save as the URL.

How can I do this? Please, can you help?

Thank you!

**Comment by valadas** [comment]
You could do a join on the TabId and the TabPath from the Tabs table potentially and then you can get the portal alias too by using the portalId of that tab and the PortalAliases table...

**Comment by maduranga001** [comment]
But the problem is users have three options. Select a URL, File, or a DNN page. This happens for selecting a DNN page. If we can save the URL of the selected DNN page, then we can use this control. Currently, I use a textbox and we have to copy the DNN page link or external website URL manually.

This happens for the image control too. If we browse an image from the DNN file system using the DNN file browser window, it saves the image ID instead of the image path.

Thanks!
